nautobot/utilities/utils.py

```diff
@@ -187,6 +188,17 @@ def serialize_object_v2(obj):
     return data
 
 
+def model_to_dict(obj, dotted_path=None):
+    """
+    Convert a Django model instance to a dictionary that is JSON serializable, leveraging a Django serializer.
+    """
+    if dotted_path:
+        # By default serialize_object_v2 will find a serializer, this is used to send in the serializer
+        # you would prefer, via a `import_string` dotted path to the serializer
+        return json.loads(JSONRenderer().render(import_string(dotted_path)(obj, context={"request": None}).data))
```

Comment:
Why not just `return import_string(dotted_path)(obj, context={"request": None}).data` - why the round trip through JSON and back?